draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2810,31 +2813,12 @@ Sun Nov  6 08:49:37 1994         ; ANSI C's asctime() format
    dynamic but the client knows the certificate that the server will
    present.) In such cases, it is important to narrow the scope of
    acceptable certificates as much as possible.
+</t>
+<t>
```

Comment:
Is the intent of the above/below paragraph to capture certificate bypass cases (the paragraph about user oriented clients already discusses this) or something else? I ask because it seems odd to devote so much text to this "application just does whatever" case. Whether it ignores the identity or checks for some random thing, the client is disavowing any of the standard security properties anyway, so there's not much we can say. Perhaps:

> Some applications may override these certificate checks in non-standard ways, such as checking for an externally-configured certificate or ignoring the identity altogether. This may leave the connection open to active attack and has security properties outside the scope of this document.

Reply:
I think this paragraph has a long history behind it, with all imaginable cases in mind. Perhaps it makes more sense with the other paragraphs rewritten.